app/src/main/java/org/simple/clinic/forgotpin/confirmpin/ForgotPinConfirmPinScreenController.kt

```diff
@@ -51,4 +53,16 @@ class ForgotPinConfirmPinScreenController @Inject constructor(
     return events.ofType<ForgotPinConfirmPinScreenBackClicked>()
         .map { { ui: Ui -> ui.goBack() } }
   }
+
+  private fun showPinMismatchedErrors(events: Observable<UiEvent>): Observable<UiChange> {
+    val previouslyEnteredPin = events.ofType<ForgotPinConfirmPinScreenCreated>()
+        .map { it.pin }
+        .cache()
```

Comment:
Whoa why do we need `cache`? Doesn't `withLatestFrom()` already cache the latest value?

Reply:
This method is currently unused and will be removed in the next PR. But yeah, you're right.